rpc_jobs/rpc_gating.yml

```diff
@@ -52,3 +52,21 @@
           run_commenter: false
     jobs:
       - 'Pull-Request-Whisperer_{repo}'
+
+- project:
+    name: 'rpc-gating-checkmarx'
+    jira_project_key: "RE"
+    scan_type:
+      - default
+      - pci
+    trigger:
+      - PM
+    scan_repo_name:
```

Comment:
Is there a benefit to prepending with `scan_`? I'd suggest keeping the same as the other standard jobs i.e. `repo_name`, `repo_url`, `branch`.

Reply:
Was trying to make it clear that the parameter relates to the repo being scanned, not the rpc-gating repo. There are often multiple repos and just having "repo" can be ambiguous. I do see that the lack of consistency is also a problem though :(

Reply:
Done